### PR TITLE
usb-to-ethernet and usb-to-serial section

### PR DESCRIPTION
**Pull request by cotosso**
document usb-to-ethernet and usb-to-serial availability and usage

References:
- https://github.com/NethServer/nethsecurity/issues/711
- https://github.com/NethServer/nethsecurity/issues/710

### CODE REVIEW

File: migration.rst [hunk 1]
```diff
@@ -159,3 +159,11 @@ The following features are not migrated to NethSecurity:
 - Bandwidth monitor (ntopng)
 - Fail2ban, it is replaced by Threat shield :ref:`brute force attempt block feature <brute_force-section>`
 - Threat shield DNS, currently :ref:`available only from command line <threat_shield_dns-section>`
+
+USB-to-Ethernet adapters
+========================
+It may rarely happen that the NethSecurity7 being migrated has a USB to Ethernet adapter connected to add a network device. These adapters should not be used in a firewall and are **not supported on NethSecurity8**. However, it is possible to install certain specific drivers for experimental purposes, not for production environments. These drivers might be useful for temporarily managing the migrated firewall while awaiting hardware with all the necessary network cards. More information can be found in the :ref:`network section <network-section>`.
```
Comment (gsanchietti) [inline migration.rst:165]: Replace:
- "NethSecurty7" -> "NethServer 7"
- "NethSecurity8" -> "NethSecurity 8"

File: migration.rst [hunk 2]
```diff
@@ -159,3 +159,11 @@ The following features are not migrated to NethSecurity:
...
+
+.. warning::
+
+  If you are using these adapters, remember that they will not work until the correct driver is installed. Keep in mind that NethSecurity8 may not have the correct driver for the adapter you are using on NethSecurity7. In this case, you will need to use a different adapter.
```
Comment (gsanchietti) [inline migration.rst:169]: Replace:
- "NethSecurty7" -> "NethServer 7"

File: network.rst [hunk 3]
```diff
@@ -130,3 +130,79 @@ First, make sure your ISP supports DHCPv6-PD, than follow these steps:
...
+  kmod-usb-net-rtl8152 - 5.15.162-1 - Kernel module for USB-to-Ethernet Realtek 8152 USB2.0/3.0 convertors
+  kmod-usb-net-smsc95xx - 5.15.162-1 - Kernel module for SMSC LAN95XX based devices
+  kmod-usb-net-sr9700 - 5.15.162-1 - Kernel module for CoreChip-sz SR9700 based USB 1.1 10/100 ethernet devices
+
+* Install the right driver:
+::
+
+  opkg install kmod-usb-net-rtl8150
+
+* Verify a new ethX interface appears using ifconfig -a
```
Comment (gsanchietti) [inline network.rst:186]:
```suggestion
* Verify a new ethX interface appears using ``ifconfig -a``
```

File: network.rst [hunk 4]
```diff
@@ -130,3 +130,79 @@ First, make sure your ISP supports DHCPv6-PD, than follow these steps:
...
+* Configure the ethernet from the UI
+
+USB-to-Serial Adapters
```
Comment (gsanchietti) [inline network.rst:189]: Not sure this is the right place: USB-to-Serial are not related to networking.

**Review by gsanchietti — CHANGES_REQUESTED** [review]
Also remember to add a note that installed packages/kernel modules are not preserved across image upgrades.

**Comment by cotosso** [comment]
> Also remember to add a note that installed packages/kernel modules are not preserved across image upgrades.

information added

File: network.rst (hunk 4 above)
Comment (cotosso) [reply network.rst]: > Not sure this is the right place: USB-to-Serial are not related to networking.

moved to remote_access section

File: remote_access.rst [hunk 5]
```diff
@@ -226,3 +226,46 @@ The keyboard layout configuration can be saved by writing the keymap code inside
...
+
+USB-to-Serial Adapters
+----------------------
+
+In case of need, NethSecurity can be used to access another server via the serial console. If the hardware does not have an RS-232 port, USB-to-serial adapters can be used. 
+For this reason, it is possible to download and install drivers for the most common adapters on NethSecurity. These drivers are provided as-is and are **not supported by Nethesis** (if using an Enterprise or Subscription version).
+
+.. warning::
+
+ These packages/kernel modules are not preserved across image upgrades, so in the event of an upgrade, you will need to download and install them again if needed.
```
Comment (gsanchietti) [inline remote_access.rst:253]:
```suggestion
 Extra packages, including kernel modules, are not preserved across image upgrades, so in the event of an upgrade, you will need to download and install them again if needed.
```

File: network.rst [hunk 6]
```diff
@@ -130,3 +130,62 @@ First, make sure your ISP supports DHCPv6-PD, than follow these steps:
...
+USB-to-Ethernet Adapters
+------------------------
+
+USB-to-Ethernet adapters are not considered suitable for use in a firewall device that is critical to network communication, for this reason the drivers are not included in the NethSecurity image.
+Only for experimental purposes, specific drivers can be installed via the package manager for use in a test environment.
+
+.. note::
+
+ It is strongly recommended **not to use these adapters in production environments**.
+ For Enterprise/Subscription versions: USB-to-Ethernet adapters **are not covered by Nethesis support**.
```
Comment (gsanchietti) [inline network.rst:143]:
```suggestion
It is strongly recommended **not to use these adapters in production environments**.
If the unis has an Enterprise or Community subscription, bear in mind that USB-to-Ethernet adapters **are not covered by Nethesis support**.
```

File: network.rst [hunk 7]
```diff
@@ -130,3 +130,62 @@ First, make sure your ISP supports DHCPv6-PD, than follow these steps:
...
+
+.. warning::
+
+ These packages/kernel modules are not preserved across image upgrades, so in the event of an upgrade, you will need to download and install them again if needed.
```
Comment (gsanchietti) [inline network.rst:147]:
```suggestion
 Extra packages, including kernel modules, are not preserved across image upgrades, so in the event of an upgrade, you will need to download and install them again if needed.
```
```

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**